app/views/shared/activities/_activity.html.haml

```diff
@@ -33,8 +33,8 @@
         - unless activity_presenter.is_oda.nil?
           = t("summary.label.activity.is_oda.#{activity_presenter.is_oda}")
       %dd.govuk-summary-list__actions
-        - if policy(activity_presenter).update? && activity_presenter.requires_is_oda?
-          = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:is_oda)}"), activity_step_path(activity_presenter, :is_oda), t("activerecord.attributes.activity.is_oda"))
+        - if policy(activity_presenter).update? && activity_presenter.requires_is_oda? && activity_presenter.is_oda.blank?
```

Comment:
I wonder if we could achieve the same result by redefining what `requires_is_oda?` means. Currently it checks that the activity is a programme, and that it's ISPF funded. We _could_ add `is_oda.nil?` to those checks, and then it will also apply to the form step (`skip_step unless @activity.requires_is_oda?` in activity_forms_controller).